docs/workforce-management-framework/topics/pro-keycloak-enablement.adoc

```diff
@@ -24,29 +28,117 @@ docker run -p 8080:8080 feedhenry/raincatcher-keycloak
 
 For more information about the *{Keycloak}* Docker Image, see the *{Keycloak}* link:{WFM-RC-KeycloakDockerURL}{WFM-RC-Branch}/README.md[README.md].
 
+IMPORTANT: The {Keycloak} realm setup included in this docker image is for demo purposes only. Ensure that you provide proper client 
+configurations for production environments. For more information about client configurations, see the {Keycloak} link:{WFM-RC-KeycloakURL}server_admin/topics/clients.html[clients documentation]
+
+[id='{context}-import-keycloak-realm-settings']
+[discrete]
+== Importing the {Keycloak} Realm Settings
+
+NOTE: This step should only be done if you choose to run your own {Keycloak} instance. The {Keycloak} Docker image
+comes with a demo RainCatcher realm setup therefore it does not need to be manually imported.
+
+. Download the link:{WFM-RC-KeycloakDockerURL}{WFM-RC-Branch}/data_files/raincatcher-realm.json[{PROJECT_NAME} realm settings]
+. Login to the {Keycloak} admin console. See xref:{context}-accessing-the-keycloak-admin-console[Accessing the {Keycloak} Admin Console] for more information.
+. Click 'Add Realm'
+. Import the link:{WFM-RC-KeycloakDockerURL}{WFM-RC-Branch}/data_files/raincatcher-realm.json[{PROJECT_NAME} realm settings] you downloaded
+. Click 'Create'
+
+This will create the demo RainCatcher realm which includes the setup for three different clients: raincatcher-portal, raincatcher-mobile and raincatcher-cloud.
+
+IMPORTANT: The provided link:{WFM-RC-KeycloakDockerURL}{WFM-RC-Branch}/data_files/raincatcher-realm.json[{PROJECT_NAME} realm settings] is only used for
+demo purposes. Ensure that you provide proper client configurations for production environments. For more information 
```

Comment:
So where is the production one? :) 
Best way to say is that by default Realm is configured with `sample users`. See bellow for guide that shows how to configure solution to use different `User Federations`  etc.